src/Rezzza/AliceExtension/Doctrine/ORMPersister.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Rezzza\AliceExtension\Doctrine;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Nelmio\Alice\ORM\Doctrine;
+
+class ORMPersister extends Doctrine
+{
+    protected $regristry;
+    protected $flush;
+
+    public function __construct(ManagerRegistry $registry, $doFlush = true)
+    {
+        $this->registry = $registry;
+        $this->flush    = $doFlush;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function persist(array $objects)
+    {
+        $managersToFlush = [];
+
+        foreach ($objects as $object) {
+            $managerName = $this->findManagerForClass(get_class($object));
+            $this->registry->getManager($managerName)->persist($object);
+
+            $managersToFlush[$managerName] = $managerName;
+        }
+
+        if ($this->flush) {
```

Comment:
Is it really what we want ? Flush after all persist operations ?

Reality is not like that. The possibility to flush before each step looks better for me. Less magic, and closer to the reality